process/statistics/statistics.go

```diff
@@ -13,18 +15,22 @@ const (
 	shardTpsDocIDPrefix = "shard"
 )
 
-var log = logger.GetOrCreate("indexer/process/generalInfo")
+var log = logger.GetOrCreate("indexer/process/statistics")
 
-type infoProcessor struct {
+type statisticsProcessor struct {
 }
 
-// NewGeneralInfoProcessor will create a new instance of general info processor
-func NewGeneralInfoProcessor() *infoProcessor {
-	return &infoProcessor{}
+// NewStatisticsProcessor will create a new instance of general statisticsProcessor
```

Comment:
`NewStatisticsProcessor will create a new instance of a statisticsProcessor`

Reply:
done.